package.json

```diff
@@ -13,7 +13,7 @@
     "build": "npm run build:all_debug & npm run build:all",
     "build:all_debug": "make all-debug",
     "build:all": "make all",
-    "build:release": "make build/libv86.js build/v86.wasm build/v86-fallback.wasm",
+    "build:release": "make build/libv86.js build/v86.wasm build/v86-fallback.wasm build/capstone-x86.min.js build/libwabt.js",
```

Comment:
These are only for debugging, I don't think they should be included in a release build.

Reply:
the bundlers will parse the debug.js requires by defaults and would try to bundle those - if we don’t distribute them the bundlers would complain unless we add explicit externs, my thought here was is to make bundling succeed on defaults, but we can also do something so that the compiled code does not contain require() on those two libraries.

Reply:
I see. If you add `if(!DEBUG) return;` to the beginning of the functions that call `require` that should cause closure compiler to remove those calls. Alternatively, I'd fine with removing the require call entirely. I haven't used those libraries from node in a long time anyway.